src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -111,102 +117,86 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/index';
-@import '../../scss/tools/index';
-
-$ec-checkbox-default-border-color: $level-4-interactive-elements !default;
-$ec-checkbox-default-background-color: $white !default;
-$ec-checkbox-checked-background-color: $level-4-tech-blue !default;
-$ec-checkbox-checked-background-hover-color: $level-3-hover !default;
-$ec-checkbox-error-color: $color-error !default;
-$ec-checkbox-disabled-color: $level-6-disabled-lines !default;
-
+<style>
 .ec-checkbox {
   &__input {
-    @include ec-screen-reader-only;
+    @apply tw-sr-only;
   }
 
   &__label-checkbox-wrapper {
-    display: flex;
-    flex-wrap: nowrap;
+    @apply tw-flex tw-flex-no-wrap;
   }
 
   &__label {
-    @include input-label;
-
-    flex-grow: 1;
-    min-width: 0;
+    @apply tw-input-label;
+    @apply tw-flex-grow;
+    @apply tw-min-w-0;
   }
 
   &__check-icon-wrapper {
-    cursor: pointer;
-    background: $ec-checkbox-default-background-color;
-    width: 20px;
-    height: 20px;
-    flex-shrink: 0;
-    border: 2px solid $ec-checkbox-default-border-color;
-    position: relative;
-    margin-right: 8px;
+    background: white;
```

Comment:
This could be `@apply tw-bg-light`

Reply:
To be honest, I'm not sure this should be so, since this color, before the migration, was defined in the variable `$ec-checkbox-default-background-color`, which had as value `$white`, and the white color should always be `white`, I think

Reply:
Nope, white color should not be treated as a special color. We have branding now, every color should be changeable. I wanted to get rid of the naming of "$white", so it can be tweaked to any color we want. I couldn't find better name than `light` for `$white` and `dark` for `$black`.
If you have:
```
.my-component {
  background-color: var(--some-color); // blue for example
  color: white;
}
```
and then in branding I'll change `--some-color` to `#fff`, you will get white text on white background.

Reply:
I think better names for them would be `key-7` and `key-8`. Because there are nothing but regular colors we use.

Reply:
Understood. Thank you for the explanation, it make totally sense.

Reply:
@online I have a suggestion here: I'd completely drop $white/$black and `ec-color-light/ec-color-dark` and go for these two colors instead:
```
--ec-gray-color-level-7: #F4F5F6
--ec-gray-color-level-1: #1C2022
```
because:
1. F4F5F6 - This is very close to white, we use it for the background of the body.
2. 1C2022 - We don't use black directly anywhere, designers don't use black color at all (it's too dark). We need black only to be a base color for shadows and backdrops (e.g `rgba(0,0,0, 0.5)`) so it doesn't matter if the base color is little bit lighter than black.

This basically means we would have no "color" meaning in the variable names, e.g. imagine if we want to create night mode of EBO, I still have to do this:
```
:root {
  --ec-color-light: black; // or something very dark
  --ec-color-dark: white; // or something very light
}
```
After dropping these we would do:
```
:root {
  --ec-gray-color-7: black; // or something very dark
  --ec-gray-color-1: white; // or something very light
}
```
Which is absolutely fine.

WDYT?

Reply:
Makes sense to me, let's add these to the gray variants :+1:

Reply:
ooof, that's too much change. I think we have to go for `gray-8`, `gray-9/0` then.